tools/tasks/seed/app.through.ts

```diff
@@ -0,0 +1 @@
+import '../../../src/client/app/main';
```

Comment:
I got it. Can you pass `src/client/tsconfig.json` as argument of `ng-xi18n`? For instance in ngc it'll be: `ngc -p src/client/tsconfig.json`.